PR: https://github.com/odoo/odoo/pull/

From: 160001ca68081bedf60b734b8391905bf0d71d7a
From: Sanjay Jamod
Date: 2017-09-05 11:29:16

Structural Changes: 4
Total Changes: 6

[IMP] mrp: Improve settings labels and installable modules

Purpose
=======

Improve settings of manufacturing app

Specification
=============

https://drive.google.com/a/odoo.com/file/d/0B0ITmdw6h3ZrMndPb2ZWbUY2Tm8/view?usp=drivesdk

================================= pseudo patch: =================================

--- a/addons/mrp/models/res_config_settings.py
+++ b/addons/mrp/models/res_config_settings.py
@@ -11,6 +11,10 @@ class ResConfigSettings(models.TransientModel):
     use_manufacturing_lead = fields.Boolean(string="Default Manufacturing Lead Time", oldname='default_use_manufacturing_lead')
     module_mrp_byproduct = fields.Boolean("By-Products")
     module_mrp_mps = fields.Boolean("Master Production Schedule")
+    module_mrp_plm = fields.Boolean("Product Lifecycle Management (PLM)")
+    module_mrp_maintenance = fields.Boolean("Maintenance")
+    module_quality_mrp = fields.Boolean("Quality")
+    module_mrp_repair = fields.Boolean("Repair")
     group_mrp_routings = fields.Boolean("Work Orders",
         implied_group='mrp.group_mrp_routings')
 
@@ -30,4 +34,4 @@ class ResConfigSettings(models.TransientModel):
     @api.onchange('use_manufacturing_lead')
     def _onchange_use_manufacturing_lead(self):
         if not self.use_manufacturing_lead:
-            self.manufacturing_lead = 0.0
\ No newline at end of file
+            self.manufacturing_lead = 0.0
